driver-core/src/main/com/mongodb/client/model/search/SearchOperator.java

```diff
@@ -292,6 +293,31 @@ static GeoNearSearchOperator near(final Point origin, final Number pivot, final
                 .append("pivot", notNull("pivot", pivot)));
     }
 
+    /**
+     * Returns a {@link SearchOperator} that returns documents similar to input documents.
+     *
+     * @param like The BSON document that is used to extract representative terms to query for.
+     * @return The requested {@link SearchOperator}.
+     * @mongodb.atlas.manual atlas-search/morelikethis/ moreLikeThis operator
+     */
+    static MoreLikeThisSearchOperator moreLikeThis(final BsonDocument like) {
+        return moreLikeThis(singleton(notNull("like", like)));
+    }
+
+    /**
+     * Returns a {@link SearchOperator} that returns documents similar to input document.
```

Comment:
I think I put the suggestion on the wrong method, and the plural might be backwards on these (this should have the s, the other, not).